client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -127,15 +127,15 @@ const NotificationSubscriptions = createReactClass( {
 						</FormFieldset>
 
 						<FormFieldset>
-							<FormLegend>{ this.props.translate( 'Jabber Subscription Delivery' ) }</FormLegend>
+							<FormLegend>{ this.props.translate( 'Jabber subscription delivery' ) }</FormLegend>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.